bot/cogs/base_converter_cog.py

```diff
@@ -0,0 +1,129 @@
+import discord
+import discord.ext.commands as commands
+
+from bot.consts import Colors
+
+class BaseConverterCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.group()
+    async def baseconvert(self, ctx):
```

Comment:
we should make this group an "invoke without subcommand, see the roles cog group command for an example, then calling just !baseconvert should list the options and their syntax

Reply:
A better name here would also probably be just convert